lib/players/human.rb

```diff
@@ -0,0 +1,9 @@
+module Players
+  class Human < Player
+
+    def move(board)
+      input = gets.chomp
+    end
```

Comment:
are you prompting the user here, such as `puts "Make your move"`?  Also, no need to assign a new variable called `input` here.  Using `gets.chomp` alone will implicitly return the user's input.